runtime/ptrace_check.cc

```diff
@@ -98,6 +110,39 @@ void EnsureDebuggedIOS(const flutter::Settings& vm_settings) {
   {
     FML_LOG(ERROR) << warning;
   }
+
+  return true;
+}
+
+static bool EnableTracingIfNecessaryOnce(const Settings& vm_settings) {
```

Comment:
Why does the name have once in it? Nothing here enforces the once-ness here it seems.

Reply:
Never mind, seems like the name is reflects the expectation rather than the implementation...

Reply:
I was following the usual C convention of of calling a method that is supposed to be called when a mutex is held with the `Locked` suffix.